update-orphaning/index.html

```diff
@@ -430,120 +448,55 @@
         }, 2000);
       }
 
+      var latestVersion = data[3]["latest-version"];
       // Populate version distribution pie chart
       {
         var orphanedVersions = data[4];
         if (!orphanedVersions || !orphanedVersions["orphaned-by-versions"]) {
           document.getElementById("version-distribution-chart-title").style.visibility = "hidden";
           document.getElementById("version-distribution-chart").style.visibility = "hidden";
         } else {
+          // Color-blind-friendly palette taken from:
+          // http://mkweb.bcgsc.ca/biovis2012/
+          var versionColors = ["#92B600",  // green
+                               "#24FF24",  // light green
+                               "#B6DBFF",  // light blue
+                               "#004949",  // dark green
+                               "#490092",  // dark purple
+                               "#924900",  // brown
+                               "#009292",  // dark turquoise
+                               "#FFB677",  // orange
+                               "#B66DFF",  // purple
+                               "#006D6D",  // darker turquoise
+                               "#6DB6FF",  // blue
+                               "#FF6DB6",  // pink
+                               "#DBD100",  // dark yellow
+                               "#FFFF6D",  // yellow
+                               "#000000"]; // black
           orphanedVersions = orphanedVersions["orphaned-by-versions"];
-          var versionDistributionObject = [];
-          var versionColors = {
-            "42": "#FF0000",
-            "43": "#FF8000",
-            "44": "#000099",
-            "45": "#FF00FF",
-            "46": "#99004C",
-            "47": "#A0A0A0",
-            "48": "#CCCC00",
-            "49": "#000066",
-            "50": "#0000FF",
-            "51": "#9933FF",
-            "52": "#009900",
-            "53": "#FFE5CC",
-            "54": "#663300",
-            "55": "#006666",
-          }
+          var versionSlices = [];
           for (v in orphanedVersions) {
-            var color = "#0255d1";
-            if (v.split(".") && v.split(".")[0] &&
-                versionColors[v.split(".")[0]]) {
-              color = versionColors[v.split(".")[0]];
+            var version = v.split(".")[0];
+            var diff = latestVersion - version;
+            var index = versionColors.length - 1;
+            if (diff < versionColors.length) {
+              index = diff;
             }
-            var slice = {
-                          "label": v,
-                          "value": orphanedVersions[v],
-                          "color": color
-                        };
-            versionDistributionObject.push(slice);
+            var color = versionColors[index];
+            var slice = [v, orphanedVersions[v], color];
+            versionSlices.push(slice);
```

Comment:
Since you're generating slices here, I see no reason not to build them straightaway into d3 format